### DISCUSSION TITLE
添加Mod文件夹中的Mod与游戏版本兼容检测，点击更新自动下载当前Minecraft版本的最新Mod

### DISCUSSION THREAD

**Discussion by SingoLee** [Meloong-Git/PCL#4300] [提问] [answered]
目前1.21的Mod最新版放入1.20.4的文件夹不会检测是否版本兼容，我希望添加自动检测是否兼容，点击Mod更新自动下载当前版本支持的最新版本mod，而不是没有更新按钮，方便制作原版整合包

**Comment by allMagicNB** [comment]
？原版整合包你要整合什么，资源包？

**Reply by SingoLee** [reply]
masa与地毯那种的生电整合包

**Reply by allMagicNB** [reply]
> masa与地毯那种的生电整合包

~~那为什么叫原版整合包~~

**Reply by Deep-Dark-Forest** [reply]
> masa与地毯那种的生电整合包

这已经装模组了，不能算原版

**Comment by WForst-Breeze** [comment, marked as answer]
参考 #728 #2824 需求相近（均为需求将旧版本 Mod 一键更新为匹配当前游戏版本的 Mod 版本）
